client/app/components/dashboards/AddWidgetDialog.jsx

```diff
@@ -103,7 +103,16 @@ class AddWidgetDialog extends React.Component {
       // which results are matching current search term and ignore
       // outdated results.
       if (this.state.searchTerm === term) {
-        this.setState({ searchedQueries: results.results });
+        const searchedQueries = results.results;
+        searchedQueries.forEach((query) => {
+          if (query.is_draft === true) {
+            query.name += ' <div class="label label-tag-unpublished">Unpublished</div>';
+          }
+          query.tags.forEach((tag) => {
+            query.name += ' <div class="label label-tag">' + tag + '</div>';
+          });
```

Comment:
@kravets-levko do you think we can use the TagsControl component here?

Reply:
I think we *should* use it here